fastlane.gemspec

```diff
@@ -102,7 +102,7 @@ Gem::Specification.new do |spec|
 
   # Development only
   spec.add_development_dependency('rake')
-  spec.add_development_dependency('rspec', '~> 3.9.0')
+  spec.add_development_dependency('rspec', '~> 3.10.0')
```

Comment:
Note: This is to use `include(...).once` matcher.

https://github.com/fastlane/fastlane/pull/17957#discussion_r557052834